src/AzureAuth/EnvVars.cs

```diff
@@ -23,6 +23,16 @@ internal static class EnvVars
         /// </summary>
         public static readonly string NoUser = $"{EnvVarPrefix}_NO_USER";
 
+        /// <summary>
+        /// Name of the env var for the Azure DevOps pipelines default personal access token.
+        /// </summary>
+        public static readonly string SystemAccessToken = "SYSTEM_ACCESSTOKEN";

```

Comment:
We don't prefix this because this is the default AT from ADO?

Reply:
Yeah, exactly. This is an env var that's defined outside of AzureAuth, but that we respect because of how common it is.